presto-main/src/main/java/com/facebook/presto/operator/aggregation/PrecisionRecallAggregation.java

```diff
@@ -182,16 +183,16 @@ public BucketResult(
         }
 
         final double totalTrueWeight = Streams.stream(state.getTrueWeights().iterator())
-                .mapToDouble(c -> c.weight)
+                .mapToDouble(c -> c.getWeight())
```

Comment:
nit: `.mapToDouble(FixedDoubleHistogram.Bucket::getWeight)`

Reply:
Changed. 

Out of curiosity, what is the advantage?

Reply:
Just coding style; we usually prefer syntactical sugar than imperative procedure if the performances are the same